Makefile

```diff
@@ -1,9 +1,10 @@
 ELVISH_MAKE_BIN ?= $(shell go env GOPATH)/bin/elvish
+ELVISH_PLUGINS ?= 0
 
 default: test get
 
 get:
-	export CGO_ENABLED=0; \
+	export CGO_ENABLED=$(ELVISH_PLUGINS); \
```

Comment:
The `go build` line below also needs to be updated to set `buildinfo.Reproducible=false` when cgo is enabled; cgo builds are in general not reproducible. You can leave this as is and I can handle that change.